airbyte-integrations/connectors/source-looker/source_looker/streams.py

```diff
@@ -251,10 +251,40 @@ def get_parent_endpoints(self) -> List[SwaggerParser.Endpoint]:
     def name(self) -> str:
         return self._name
 
+    @classmethod
+    def format_null_in_schema(cls, schema: Mapping[str, Any]):
```

Comment:
could you add a small unit test for this please?

Reply:
updated